README.md

```diff
@@ -1120,6 +1121,27 @@ Options for the [OpenID Connect](http://developer.okta.com/docs/api/resources/oi
     }
     ```
 
+### Smart Card IdP
+
+Options for authentication with a Smart Card `X509` type IdP.
+
+- `certAuthUrl` - Required url endpoint that requests the browser for a certificate.
```

Comment:
we can probably omit "Required" and just add "Optional" in the ones we need. This is similar to other configs we have. 
nit: we should uppercase URL
```
`certAuthUrl` - URL endpoint that requests the browser for a certificate.
`text` *(optional)* - 
`className ` *(optional)* - 
etc
```
same comment for the other optional params